plugins/simpleadminhacks-paper/src/main/java/com/programmerdan/minecraft/simpleadminhacks/framework/utilities/BuildLimit.java

```diff
@@ -0,0 +1,81 @@
+package com.programmerdan.minecraft.simpleadminhacks.framework.utilities;
+
+import org.bukkit.configuration.serialization.ConfigurationSerializable;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public class BuildLimit implements ConfigurationSerializable {
```

Comment:
If this class will only be used by the BuildLimits hack, it could be made an inner-class of the hack. That's just a personal preference though.

Reply:
SoundTech OK'ed this so :man_shrugging:

Reply:
Okay :+1: